google/cloud/storage/ci/run_integration_tests_emulator_bazel.sh

```diff
@@ -91,4 +90,9 @@ testbench_args=(
   "${excluded_targets[@]}"
 exit_status=$?
 
+if [[ "$exit_status" -ne 0 ]]; then
+  source "${PROJECT_ROOT}/ci/define-dump-log.sh"
+  dump_log testbench.log
```

Comment:
Is it the right location of `testbench.log` ?

Reply:
Ooops, no.  I think it is placed in `$HOME`, see above just before `start_testbench`.

Reply:
Could I fix it in the next PR ?

Reply:
Yes, absolutely, I thought you needed it to troubleshoot something.

Reply:
This time, no test failed so I cannot troubleshoot anything. Just to be sure, the extract location of `testbench.log` is `$HOME/testbench.log` right ?